git/util.py

```diff
@@ -52,7 +52,18 @@
            'RemoteProgress', 'CallableRemoteProgress', 'rmtree', 'unbare_repo',
            'HIDE_WINDOWS_KNOWN_ERRORS')
 
+
+try:  # Python 2.7+
+    import logging.NullHandler
+except ImportError:
+    class NullHandler(logging.Handler):
```

Comment:
I don't understand what the NullHandler is used for. So far it was sufficient to obtain a logging using `logging.getLogger('git.util')` and assume parties interested in logging have set it up to contain/inherit handlers.